src/test/java/com/Timos/TicTacToeInputProducer/TicTacToeInputProducerApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package com.Timos.TicTacToeInputProducer;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class TicTacToeInputProducerApplicationTests {
```

Comment:
Appreciably you're just getting it up and running and testing the Kafka config will require some wiremocking, but make sure when you get moving that you start adding tests